solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,190 @@
+
+    Returns:
+        string: Updated password if the choice is 4, or updated id and
+            password if the choice is 5.
+    """
+    if costumer_choice == 1:
+        print(f"Balance: {costumers_data[costumer_id]['balance']}")
+
+    elif costumer_choice == 2:
+        print("Action succeeded\n" if cash_withdrawal(
+            costumer_id, costumers_data) else "Failed - check amount\n")
+
+    elif costumer_choice == 3:
+        print(
+            "Action succeeded\n" if cash_deposit(costumer_id, costumers_data)
+            else "Failed - check amount\n")
+
+    elif costumer_choice == 4:
+        new_password = input("New password: ")
+        costumer_password = new_password if change_password(
+            costumer_id, costumers_data, new_password) else costumer_password
+        return costumer_password
+
+    elif costumer_choice == 5:
+        costumer_id = input("id: ")
+        if costumer_id != '-1':
+            costumer_password = input("Password: ")
+        return costumer_id, costumer_password
+
+    else:
+        print("Please choose 1/2/3/4/5 option: ")
+
+
+def manage_costumer_actions(costumer_id, costumers_data, costumer_password):
+    """Manage costumer actions.
+
+     Args:
+        costumers_data (dictionary): Bank costumer data.
+        costumer_id (string): Costumer id.
+        costumer_password (string): Current costumer password.
+
+    Returns:
+        None.
+    """
+    while costumer_id != '-1':
+        if costumer_id in costumers_data and costumers_data[
+                costumer_id]['password'] == costumer_password:
+            try:
+                costumer_choice = int(input(COSTUMER_CHOICE_OPTIONS))
+                if costumer_choice == 4:
+                    costumer_password = check_costumer_choice(costumer_choice,
+                                                              costumers_data,
+                                                              costumer_id,
+                                                              costumer_password)
+
+                elif costumer_choice == 5:
+                    costumer_id, costumer_password = check_costumer_choice(
+                        costumer_choice, costumers_data, costumer_id,
+                        costumer_password)
+
+                else:
+                    check_costumer_choice(costumer_choice, costumers_data,
+                                          costumer_id, costumer_password)
+
+            except ValueError:
+                print("Value Error! your input needs to be digit value\n")
+
+        else:
+            print("ID or password is not correct.Try to enter again:\n")
+            costumer_id, costumer_password = input("id: "), input(
+                "Password: ")
+
+
+def main():
+    costumers_data = create_dictionary(path_of_file=sys.argv[ATM_FILE_PLACE])
+    if costumers_data is not None:
+        costumer_id, costumer_password = input("id: "), input("Password: ")
```

Comment:
Think again about the design of your code.
You read the id and password in 3 different places, is it really necessary?